packages/web3-core-method/package-lock.json

```diff
@@ -134,7 +134,7 @@
 				"fs-extra": "^6.0.1",
 				"strip-json-comments": "^2.0.1",
 				"tslint": "^5.12.0",
-				"typescript": "^3.8.0-dev.20191128"
+				"typescript": "^3.7.0-dev.20191015"
```

Comment:
Means this package-lock doesn't get updated anyways and I was misleading updating it? Are separate PRs required from your point of view to update those? (I mean all package-locks)